ludwig/backend/horovod.py

```diff
@@ -69,4 +69,16 @@ def is_coordinator(self):
 
     @property
     def num_nodes(self) -> int:
+        if not self._horovod:
+            return 1
         return self._horovod.size()
+
+    @property
+    def num_cpus(self) -> int:
+        # TODO: Implement
+        return 1
+
+    @property
+    def num_gpus(self) -> int:
+        # TODO: Implement
+        return 0
```

Comment:
@tgaddair I'm not sure what to do for HorovodBackend - any help would be great!

Reply:
Would 
- CPUs = `self._horovod.size()`
- GPUs = `self._horovod.local_rank()`
?